README.md

```diff
@@ -825,6 +847,9 @@ Please follow these steps to build and test the code:
 * Startup an Artifactory-Pro instance.
 * Set the *CLIENTTESTS_ARTIFACTORY_URL*, *CLIENTTESTS_ARTIFACTORY_USERNAME* and *CLIENTTESTS_ARTIFACTORY_PASSWORD*
   environment variables with your Artifactory URL, username and password.
+* You can also set *clienttests.artifactory.url*, *clienttests.artifactory.username*,*clienttests.artifactory.password*, 
+  and clienttests.artifactory.federationUrl(only to test federated repository which is an optional field) to run your tests locally with your artifactory url, username and password
```

Comment:
* I suggest rephrasing - 
...and clienttests.artifactory.federationUrl(only to test federated repository which is an optional field) to run your tests locally with your artifactory url, username and password

to -

...and *clienttests.artifactory.federation.url* (optional field, used for Federated Repositories testing).

* Let's rename ```clienttests.artifactory.federationUrl``` to ```clienttests.artifactory.federation.url``` so that it matches the convention.

* We should also support a new CLIENTTESTS_ARTIFACTORY_FEDERATION_URL environment varoable, and add it to the documentation with the other variables.

Reply:
I have changed the environment variable to be CLIENTTESTS_ARTIFACTORY_FEDERATION_URL  and also the clienttests.artifactory.federation_url. This is to avoid a naming conflict.